doc/src/dev_intro.rst

```diff
@@ -0,0 +1,152 @@
+MIGraphX Fundamentals
+======================
+
+MIGraphX provides an optimized execution engine for deep learning neural networks.
+We will cover some simple operations in the MIGraphX framework here.
+For a quick start guide to using MIGraphX, look in the examples directory: ``https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/tree/develop/examples/migraphx``.
```

Comment:
Not a change for this PR, but we should open an issue to move this example quick start guide to our documentation.